packages/synapse-react-client/src/components/DataGrid/StartGridSession.tsx

```diff
@@ -0,0 +1,292 @@
+import { useState, useRef, useEffect } from 'react'
+import { useCreateGridSession } from './useCreateGridSession'
```

Comment:
Probably move `useCreateGridSession` to synapse-queries folder (inside a `useGridSession.ts` file that also contains hooks to fetch the list & delete a session